src/Servers/HttpSys/test/FunctionalTests/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests.csproj

```diff
@@ -22,8 +22,8 @@
     <Compile Include="$(SharedSourceRoot)InternalHeaderNames.cs" Linkbase="shared"/>
     <Compile Include="$(KestrelSharedSourceRoot)test\TestResources.cs" LinkBase="shared" />
```

Comment:
What about these other kestrel shared sources that are being used outside of kestrel?

Reply:
I've moved remaining Kestrel shared sources (`TestResources.cs` and `\TestCertificates`) to `src\Shared`.